packages/ml-vision/ios/RNFBMLVision/RNFBMLVisionModule.m

```diff
@@ -15,24 +15,21 @@
  *
  */
 
-#import <React/RCTUtils.h>
-#import <Firebase/Firebase.h>
 
-#import "RNFBMlkitModule.h"
-#import "RNFBApp/RNFBSharedUtils.h"
+#import <React/RCTUtils.h>
+#import "RNFBSharedUtils.h"
+#import "RNFBMLVisionModule.h"
 
 
-@implementation RNFBMlkitModule
+@implementation RNFBMLVisionModule
 #pragma mark -
 #pragma mark Module Setup
 
-  RCT_EXPORT_MODULE();
-
-  - (dispatch_queue_t)methodQueue {
-    return dispatch_get_main_queue();
-  }
+RCT_EXPORT_MODULE();
 
 #pragma mark -
-#pragma mark Firebase Mlkit Methods
+#pragma mark Firebase ML Kit Vision Methods
+
+// TODO react methods here
```

Comment:
Delete this file?

Reply:
Fixed in commit I just pushed from your Mac